src/reactive-elements.js

```diff
@@ -107,11 +107,22 @@ React = typeof React === 'object' ? React : require('react');
         }
 
         return result;
-    }
+    };
 
     var parseAttributeValue = function (value) {
-        var regexp = /\{.*?\}/g;
-        var matches = value.match(regexp);
+        // Testing for json string first
+        var matches = value.match(/^\{((?:\{|\[)(?:.|[\r\n])*(?:\}|\]))}$/), result;
```

Comment:
Could you put a bit of explanation on what this regexp is looking for? Is it double curly braces sequence without : between?